File: javascript/rna-transcription/rna-transcription.js
// RNA Transcription

export const toRna = (strandRNA = '') => {
  const basePairs = { 'A': 'U', 'T': 'A', 'G': 'C', 'C': 'G' };

  let complementStrand = '';

  strandRNA.split('').map(function(am) { complementStrand += basePairs[am] });

  return complementStrand;
};

File: python/rna-transcription/rna_transcription.py
# RNA Transcription

def to_rna(dna_strand):
    nucleotides = { 'G': 'C', 'C': 'G', 'T': 'A', 'A': 'U' }

    return ''.join([nucleotides[nt] for nt in dna_strand ])
